src/test/java/hudson/scm/SubversionSCMUnitTest.java

```diff
@@ -108,6 +109,7 @@ public void shouldSetEnvironmentVariablesWithMultipleSvnModules() throws IOExcep
     private SubversionSCM mockSCMForBuildEnvVars() {
         SubversionSCM scm = mock(SubversionSCM.class);
         doCallRealMethod().when(scm).buildEnvVars(any(AbstractBuild.class), anyMapOf(String.class, String.class));
+        doCallRealMethod().when(scm).buildEnvironment(any(Run.class), anyMapOf(String.class, String.class));
```

Comment:
So does this allow

    mvn -Djenkins.version=2.60 clean test

to work? What about PCT’s

    mvn clean test-compile && mvn -Djenkins.version=2.60 surefire:test

?